frontend/src/pages/register.js

```diff
@@ -27,9 +28,11 @@ const Register = () => {
     // if there are no errors, try register through api
     if (Object.keys(errors).length === 0) {
       try {
-        await userService.register(registrant);
+        const { data } = await userService.register(registrant);
+        userService.createProfile({ user_id: data.id });
```

Comment:
create profile should be process during registration api in just 1 call. there will be a big chance that in this part will have bugs